modal/createModal.html

```diff
@@ -12,32 +12,32 @@ <h5 class="modal-title">New Contact</h5>
             <div class="modal-body">
                 <!--Info-->
                 <div class="info">
-                    <form id="editForm" class="row g-3">
-                        <div class="col-md-4">
+                    <form id="new-form" class="row g-3">
+                        <div class="col-md-4" data-contact-key="FirstName">
                             <label for="new-first-name" class="form-label">First Name</label>
                             <input name="FirstName" type="name" class="form-control" id="new-first-name">
                         </div>
-                        <div class="col-md-4">
+                        <div class="col-md-4" data-contact-key="LastName">
                             <label for="new-last-name" class="form-label">Last Name</label>
                             <input name="LastName" type="name" class="form-control" id="new-last-name">
                         </div>
-                        <div class="col-md-4">
+                        <div class="col-md-4" data-contact-key="PhoneNumber">
                             <label for="new-phone-number" class="form-label">Phone Number</label>
                             <input name="PhoneNumber" type="phone-number" class="form-control" id="new-phone-number">
                         </div>
-                        <div class="col-8">
+                        <div class="col-8" data-contact-key="Address">
                             <label for="new-address" class="form-label">Address</label>
-                            <input name="Address" type="text" class="form-control" id="new-address" placeholder="1234 Main St">
+                            <input name="Address" type="text" class="form-control" id="new-address">
                         </div>
-                        <div class="col-md-4">
+                        <div class="col-md-4" data-contact-key="ZIP">
                             <label for="new-zip" class="form-label">Zip</label>
                             <input name="ZIP" type="text" class="form-control" id="new-zip">
                         </div>
-                        <div class="col-md-8">
+                        <div class="col-md-8" data-contact-key="City">
                             <label for="new-city" class="form-label">City</label>
                             <input name="City" type="text" class="form-control" id="new-city">
                         </div>
-                        <div class="col-md-4">
+                        <div class="col-md-4" data-contact-key="State">
```

Comment:
I just saw this now, it looks like your using too much column spacing(col-md-*). In addition columns should only be in row div elements (which then should be stored in a container div). With bootstrap, the column lengths should add up to 12 exactly per row. As it stands now it's adding up to 36. This should probably be fixed for all the modals.

Reply:
The bootstrap-5 website mentions that if you are doing full column coverage all the way down a row, it is ok to simply stack multiples of 12. They do it several times in their examples. We can change if you want, but it will just add extra visual clutter imo

Reply:
TIL that! Disregard my previous comment

Reply:
Awesome! Yeah I was kinda surprised when I saw them doing it in the examples but if they recommend it then why not!!